tests/blocks.rs

```diff
@@ -119,3 +119,67 @@ async fn test_blocks() {
 
     db.drop().await.unwrap();
 }
+
+#[tokio::test]
+async fn test_milestone_activity() {
+    let db = connect_to_test_db("test-milestone-activity").await.unwrap();
+    db.clear().await.unwrap();
+    let collection = db.collection::<BlockCollection>();
+    collection.create_indexes().await.unwrap();
+
+    // Note that we cannot build a block with a treasury transaction payload.
+    let blocks = vec![
+        get_test_transaction_block(),
+        get_test_transaction_block(),
+        get_test_milestone_block(),
+        get_test_tagged_data_block(),
+        get_test_no_payload_block(),
+    ]
+    .into_iter()
+    .enumerate()
+    .map(|(i, block)| {
+        let bee_block = bee::Block::try_from(block.clone()).unwrap();
+        let parents = block.parents.clone();
+        (
+            bee_block.id().into(),
+            block,
+            bee_block.pack_to_vec(),
+            BlockMetadata {
+                parents,
+                is_solid: true,
+                should_promote: false,
+                should_reattach: false,
+                referenced_by_milestone_index: 1.into(),
+                milestone_index: 0.into(),
+                inclusion_state: match i {
```

Comment:
Not sure there's any point in using anything but `Included` for txns and `NoTransaction` for everything else.

Reply:
huh...not every tx gets included